api/src/main/java/org/openmrs/module/eptsreports/reporting/library/indicators/PregnantIndicators.java

```diff
@@ -18,9 +18,8 @@
 	 * @return CohortIndicator
 	 */
 	public CohortIndicator getPregnantWomenWithSuppressedViralLoadIn12Months() {
```

Comment:
Same here - these all look the same. I think we can merge into one generic indicator and pass in the name and the cohort definition as parameters